packages/contracts-bsc/contracts/common/BurnableTokenWithBounds.sol

```diff
@@ -38,7 +38,7 @@ abstract contract BurnableTokenWithBounds is ReclaimerToken {
      * - `msg.sender` must have at least `amount` tokens.
      *
      */
-    function burn(uint256 amount) external override {
+    function burn(uint256 amount) external {
```

Comment:
Let's either inherit `ITrueCurrency` here or add a new `IBurnableTokenWithBounds` interface to inherit.